autoware_lanelet2_map_validator/src/common/validation.cpp

```diff
@@ -372,4 +377,61 @@ void export_results(json & json_data, const std::string output_file_path)
   std::cout << "Results are output to " << file_path << std::endl;
 }
 
+ValidatorExclusionMap import_exclusion_list(const json & json_data)
+{
+  ValidatorExclusionMap result_map;
+  const std::vector<std::string> checks =
+    lanelet::validation::availabeChecks(".*");  // cspell:disable-line
+  const std::set<std::string> valid_primitives = {
+    "point", "linestring", "polygon", "lanelet", "area", "regulatory element", "primitive"};
+
+  for (const std::string & check : checks) {
+    result_map[check] = {};
+  }
+
+  for (const auto & object : json_data["exclusion"]) {
+    const std::string primitive = object["primitive"];
+    const lanelet::Id id = object["id"];
+
+    if (valid_primitives.find(primitive) == valid_primitives.end()) {
+      throw std::invalid_argument(
+        "Invalid primitive " + primitive + " was found in the exclusion list");
+    }
+
+    if (object.contains("validators")) {
+      for (const auto & validator : object["validators"]) {
+        std::string validator_name = validator["name"];
+        if (result_map.find(validator_name) == result_map.end()) {
+          throw std::invalid_argument(
+            "Invalid validator " + validator_name + " was found in the exclusion list");
+        }
+        result_map[validator_name].push_back({primitive, id});
+      }
+    } else {
+      for (const std::string & check : checks) {
+        result_map[check].push_back({primitive, id});
+      }
+    }
+  }
+
+  return result_map;
+}
+
+void filter_out_primitives(
+  std::vector<lanelet::validation::DetectedIssues> & issues_vector,
+  std::vector<SimplePrimitive> primitive_list)
```

Comment:
Can you rename the argument to be more comprehensive ? Like

```suggestion
  std::vector<SimplePrimitive> primitive_list_to_exclude)
```